.mvn/rrf/groupId-confluent.txt

```diff
@@ -0,0 +1,3 @@
+io.confluent
+org.apache.kafka
```

Comment:
apache kafka too?

Reply:
Given _proper repo ordering_ (and RRF), anything present on Central will come from there, and only "not on central" kafka stuff (and there are such things, try to remove this line locally and build) will be tried from confluent remote repository.

Reply:
can this be code-commented?